Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -35,7 +33,8 @@ class XcodeSpec: QuickSpec {
 			_ = try? FileManager.default.removeItem(at: targetFolderURL)
 		}
 
-		describe("determingSwiftInformation") {
+		describe("determineSwiftInformation:") {
+			let currentSwiftVersion = swiftVersion.single()?.value
 			let testSwiftFramework = "Quick.framework"
 			let currentDirectory = URL(fileURLWithPath: FileManager.default.currentDirectoryPath, isDirectory: true)
```

Comment:
Just noticed we build a fixtured Swift framework (`WorkspaceWithDependency`) later in `XcodeSpec`.

I'm gonna explore the possibility of sharing the side effects of this build with multiple `describe` and `it` calls…

And sorry about the Travis CI image thing, that was unfortunate timing, apologies.